app/generators/new_service_generator.rb

```diff
@@ -46,4 +47,47 @@ def footer_pages
       end
     end
   end
+
+  def default_form_flow_object(metadata)
+    start_page_uuid = metadata['pages'][0]['_uuid']
+    cya_page_uuid = metadata['pages'][1]['_uuid']
+    confirmation_page_uuid = metadata['pages'][2]['_uuid']
+
+    {
+      start_page_uuid => {
+        '_type': 'flow.page',
+        'next': {
+          'default': cya_page_uuid
+        }
+      },
+      cya_page_uuid => {
+        '_type': 'flow.page',
+        'next': {
+          'default': confirmation_page_uuid
+        }
+      },
+      confirmation_page_uuid => {
+        '_type': 'flow.page',
+        'next': {
+          'default': ''
+        }
+      }
+    }
+  end
+
+  def add_cya_and_confirmation(metadata)
+    cya_metadata = (DefaultMetadata['page.checkanswers'])
+    .merge('_uuid' => SecureRandom.uuid)
+    .merge('url' => 'checkanswers')
```

Comment:
little bit difficult to read this syntactically. can we indent the merges please.

also there is no need to wrap the call to `DefaultMetadata` in parenthesis